contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo

```diff
@@ -102,6 +102,26 @@ func l1_sender{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
     return (address)
 end
 
+# --- Ownership ---
+
+@view
+func owner{pedersen_ptr : HashBuiltin*, syscall_ptr : felt*, range_check_ptr}() -> (owner : felt):
```

Comment:
Looks like we are also missing these Ownership view/external fn definitions in the `proxy.cairo`, right?

Also, let's rename the `proxy.cairo` as `aggregator_proxy.cairo` as it's not a general proxy, but specifically an aggregator interface proxy.

Reply:
We could also use an `shouldBehaveLikeOwnableContract` behaviour test we could use for multiple contracts where this behaviour is expected.

Reply:
@archseer I've made these changes.